README.md

```diff
@@ -1,107 +1,52 @@
 
 As a maker
 So that I can see what others are saying  
-I want to see all peeps in reverse chronological order
+I want to see all peeps in reverse chronological order -- READ THE PEEPS ON FRONT PAGE
 
 As a Maker
 So that I can better appreciate the context of a peep
-I want to see the time at which it was made
+I want to see the time at which it was made --- WORK IN PROGRESS, NOT YET IMPLEMENTED
 
 As a Maker
 So that I can post messages on Chitter as me
-I want to sign up for Chitter
+I want to sign up for Chitter -- PROVIDE ALL REQUIRED INFO AND PRESS SIGN UP
 
 HARDER
 
 As a Maker
 So that only I can post messages on Chitter as me
-I want to log in to Chitter
+I want to log in to Chitter -- PRESS LOG IN WITH CORRECT CREDENTIALS
 
 As a Maker
 So that I can avoid others posting messages on Chitter as me
-I want to log out of Chitter
-
-ADVANCED
-
-As a Maker
-So that I can stay constantly tapped in to the shouty box of Chitter
-I want to receive an email if I am tagged in a Peep
-```
-
-Notes on functionality:
-------
-
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
-
-Bonus:
------
-
-If you have time you can implement the following:
-
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
-
-And/Or:
-
-* Work on the CSS to make it look good.
-
-Good luck and let the chitter begin!
-
-Code Review
------------
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
-
-Notes on test coverage
-----------------------
-
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
-
-```ruby
-require 'simplecov'
-require 'simplecov-console'
+I want to log out of Chitter -- PRESS LOG OUT BUTTON
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
 ```
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+## Setting up the environment
```

Comment:
Good instructions for DB migration. Should also include how to run and interact with the app, and how to run tests.

Try to use Markdown formatting to improve the readability of your README!